docs/multicast-guide.md

```diff
@@ -137,6 +137,13 @@ Support for Windows and IPv6 will be added in the future.
 Configuration option `multicastInterfaces` is not supported with encap mode.
 Multicast packets in encap mode are SNATed and forwarded to the transport interface only.
 
+### Large number of receiver groups for one node
+
+A Linux host limits the maximum number of multicast groups it can subscribe to
+by the default number of 20 [/proc/sys/net/ipv4/igmp_max_memberships](https://sysctl-explorer.net/net/ipv4/igmp_max_memberships/).
```

Comment:
Better to say:
```suggestion
A Linux host limits the maximum number of multicast groups it can subscribe to, 
the default number is 20 [/proc/sys/net/ipv4/igmp_max_memberships](https://sysctl-explorer.net/net/ipv4/igmp_max_memberships/).
```

Reply:
I think your suggestion is grammatically incorrect. `A Linux host limits the maximum number of multicast groups it can subscribe to, 
the default number is 20`  contains two independent clauses, and they are improperly joined with a comma.

Reply:
Maybe: "A Linux host limits the maximum number of multicast groups it can subscribe to; the default number is 20. The limit can be changed by setting [/proc/sys/net/ipv4/igmp_max_memberships](https://sysctl-explorer.net/net/ipv4/igmp_max_memberships/)."?

Reply:
@ceclinux you can follow @jianjuns 's suggestion to refine the sentence, the purpose is to make the sentence more readable.

Reply:
updated. Thanks